src/listr/tasks/tenderdash/initTaskFactory.js

```diff
@@ -0,0 +1,61 @@
+const { Listr } = require('listr2');
+
+/**
+ * @param {initializeTenderdashNode} initializeTenderdashNode
+ * @param {Docker} docker
+ * @return tenderdashInitTask
+ */
+function tenderdashInitTaskFactory(
+  initializeTenderdashNode,
+  docker,
+) {
+  /**
+   * @param {Config} config
+   * @return {Listr}
+   */
+  function tenderdashInitTask(
+    config,
+  ) {
+    return new Listr([
+      {
+        title: 'Generate node keys and data',
+        task: async (ctx, task) => {
+          const isValidatorKeyPresent = Object.keys(config.get('platform.drive.tenderdash.validatorKey')).length !== 0;
+          const isNodeKeyPresent = Object.keys(config.get('platform.drive.tenderdash.nodeKey')).length !== 0;
+          const isGenesisPresent = Object.keys(config.get('platform.drive.tenderdash.genesis')).length !== 0;
+
+          const { Volumes: existingVolumes } = await docker.listVolumes();
+          const { COMPOSE_PROJECT_NAME: composeProjectName } = config.toEnvs();
+          const isDataVolumePresent = existingVolumes.find((v) => v.Name === `${composeProjectName}_drive_tenderdash`);
+
+          if (isValidatorKeyPresent && isNodeKeyPresent
+            && isGenesisPresent && isDataVolumePresent) {
+            task.skip('Node already initialized');
+          } else {
```

Comment:
You can just `return` after `skip` so you don't need `else`

Reply:
OK, I changed it. But I don't understand what is being returned?

Reply:
Nothing. Just `void`.

Reply:
Why is returning `void` better than if/else? We have already evaluated the condition, so it doesn't cost any processing time and the syntax is a lot clearer (to me)

Reply:
It's just a best practice for readability.

https://medium.com/better-programming/why-you-need-to-stop-using-else-statements-5b1fd09dea9e
https://blog.timoxley.com/post/47041269194/avoid-else-return-early

Reply:
Thanks! Reading this helped my understanding a lot :+1: